### PR TITLE
Testing Ruby Code with RSpec - Tic Tac Toe

### PR DESCRIPTION
**Pull request by Flakster**
## This Pull request includes:
- Specs for all public methods in Tic Tac Toe project.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  # This option will default to `:apply_to_host_groups` in RSpec 4 (and will
+  # have no way to turn it off -- the option exists only for backwards
+  # compatibility in RSpec 3). It causes shared context metadata to be
+  # inherited by the metadata hash of host groups and examples, rather than
+  # triggering implicit auto-inclusion in groups with matching metadata.
+  config.shared_context_metadata_behavior = :apply_to_host_groups
+
+# The settings below are suggested to provide a good initial experience
+# with RSpec, but feel free to customize to your heart's content.
+=begin # rubocop:disable Style/BlockComments
```
[the remark this answers is not in the document]
Comment (Flakster) [reply spec/spec_helper.rb]: I'll leave automatically created files untouched

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi @Flakster       
#### Great Job so far, but there is one more thing required. 
- [ ] Lets write test for when there is a draw between both players.

### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Comment by Flakster** [comment]
## This new submission includes:
- New method #draw? in class Game, that checks for a draw between players
- Test for the new #draw? method

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Amazing work, Congratulation :+1: Project Approved.
You did all changes required previously. Go ahead and merge to develop.

_Goodluck in your next project_ *Happy coding*

**Pull request merged**